container/build.go

```diff
@@ -19,7 +20,15 @@ type BuildResponse struct {
 
 // Build a docker image
 func Build(path string) (tag string, err error) {
-	buildContext, err := archive.Tar(path, archive.Gzip)
+	excludeFilesBytes, err := ioutil.ReadFile(filepath.Join(path, ".mesgignore"))
```

Comment:
This is not valid because this assume that the `.mesgignore` file is required which is not true